components/form/form-native.js

```diff
@@ -0,0 +1,148 @@
+import { findFormElements, getFormElementData, isCustomFormElement } from './form-helper.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { FormMixin } from './form-mixin.js';
+import { ValidationType } from './form-element-mixin.js';
+
+/**
+ * A component that can be used to build sections containing interactive controls that are validated and submitted as a group.
+ * These interactive controls are submitted using a native HTML form submission.
+ * @slot - The native and custom form elements that participate in validation and submission
+ * @fires submit - Dispatched when the form is submitted
+ * @fires formdata - Dispatched after the entry list representing the form's data is constructed. This happens when the form is submitted.
+ */
+class FormNative extends FormMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * The URL that processes the form submission.
+			 */
+			action: { type: String },
+			/**
+			 * If the value of the method attribute is post, enctype is the MIME type of the form submission.
+			 * @type {'application/x-www-form-urlencoded'|'multipart/form-data'|'text/plain'}
+			 */
+			enctype: { type: String },
+			/**
+			 * The HTTP method to submit the form with.
+			 * @type {'POST'|'GET'}
+			 */
+			method: { type: String },
+			/**
+			 * Indicates where to display the response after submitting the form.
+			 * @type {'_self '|'_blank'|'_parent'|'_top'}
+			 */
+			target: { type: String },
+		};
+	}
+
+	get action() {
+		return this._form.action;
+	}
+
+	set action(val) {
+		this._form.action = val;
```

Comment:
What's the benefit to referencing the form's values vs. rendering the form as part of `render()` and then binding these as normal attributes on the form? Does the mixin really need to "own" the form?

Reply:
The form is special because its actually added as a child inside the slot, not rendered within `d2l-form-native`'s shadow DOM. Its done this way because native form elements need to be part of the form which is done via ID linking.

![Capture](https://user-images.githubusercontent.com/11379933/90050446-30d16d00-dca4-11ea-90d6-22e23b0daeb0.PNG)

Reply:
I've changed this so the `form` is created just-in-time during submission rather than in the `constructor`. If it causes any issues with native browser validation I can always swap it back.